requirements-dev.txt

```diff
@@ -8,10 +8,10 @@ mypy==0.930
 pytest==6.2.5
 pytest-aiohttp==0.3.0
 pytest-cov==3.0.0
+pytest-forked==1.4.0
 pytest-mock==3.6.1
 pytest-sugar==0.9.4
 pytest-timeout==2.0.2
 pytest-toolbox==0.4
-pytest-xdist==2.5.0
```

Comment:
Why don't you like xdist, by the way? Any tests using signals?

Reply:
I was under the impression it was only used for `--boxed` and the documentation suggested this functionality had been moved to pytest-forked. I don't really know anything about it.

Reply:
Right. I don't think it ever worked taking into account that it's also used in windows. FWIW `xdist` creates several "workers" and each worker executes a number of tests subsequently while `forked` creates subprocesses for every or just marked tests.

Reply:
FWIW try keeping the parallelism. Just add `--numprocesses=auto` to the config.

Reply:
> I was under the impression it was only used for `--boxed`

It was also used for `-n4` in `Makefile` but you removed those entries. This is why I suggested putting this setting into the config with `auto`.

Reply:
The tests take 3-5 seconds to run. It's not worth thinking about.